csrankings-c.csv

```diff
@@ -923,6 +923,7 @@ Christopher M. Brown,University of Rochester,https://www.cs.rochester.edu/~brown
 Christopher M. Danforth,University of Vermont,http://www.uvm.edu/~cdanfort,huQnquIAAAAJ
 Christopher M. Homan,Rochester Institute of Technology,https://www.cs.rit.edu/~cmh,kU6puLcAAAAJ
 Christopher M. Jermaine,Rice University,https://www.cs.rice.edu/~cmj4,D2P2B0MAAAAJ
+Christopher M. Reardon,University of Denver,https://ritchieschool.du.edu/about/people/christopher-m-reardon,I3L9Y6AAAAAJ

```

Comment:
This web page is a 404, faculty member not listed here: https://ritchieschool.du.edu/about/people. Please re-open with fix / to clarify.

Reply:
Website is fixed now - can you please reopen?  School CMS occasionally "updates" by deleting faculty pages and the parties that oversee this were out because of the US holiday.